services/ui-src/src/verbiage/pages/accordion.ts

```diff
@@ -10,14 +10,26 @@ export default {
         },
         {
           type: "text",
-          as: "span",
+          as: "p",
```

Comment:
A thing I learned while looking at this PR...I am not sure we need this `as: "p"` line! The parser / `<Text>` component defaults to a p tag.